InnerEye/ML/configs/histo_configs/classification/DeepSMILEPanda.py

```diff
@@ -113,7 +133,9 @@ def get_data_module(self) -> PandaTilesDataModule:
 
     def get_trainer_arguments(self) -> Dict[str, Any]:
         # These arguments will be passed through to the Lightning trainer.
-        return {"callbacks": self.callbacks}
+        kw_args = super().get_trainer_arguments()
+        kw_args["callbacks"] = self.callbacks
+        return kw_args
```

Comment:
use container get_callbacks method for simplicity

Reply:
Changed here as well, thanks.